spec/helpers/3p/markdown.js

```diff
@@ -1,26 +1,26 @@
-const Code = require('code');
-const Lab = require('lab');
-const lab = exports.lab = Lab.script();
-const expect = Code.expect;
-const it = lab.it;
-const describe = lab.describe
-
-const { buildRenderer } = require('../../spec-helpers');
-const renderer = buildRenderer();
-const hbs = renderer.handlebars;
-const helpers = require('../../../helpers/3p/markdown');
-
-Object.keys(helpers).forEach(key => {
-    hbs.registerHelper(key, helpers[key]);
-});
-
-
-describe('markdown', function() {
-  describe('markdown', function() {
-    it('should render markdown using the {{#markdown}} block helper', function(done) {
-      var template = hbs.compile('{{#markdown}}## {{../title}}{{/markdown}}');
-      expect(template({title: 'Markdown Test'})).to.equal('<h2>Markdown Test</h2>\n');
-      done();
-    });
-  });
-});
\ No newline at end of file
+// const Code = require('code');
+// const Lab = require('lab');
+// const lab = exports.lab = Lab.script();
+// const expect = Code.expect;
+// const it = lab.it;
+// const describe = lab.describe
+//
+// const { buildRenderer } = require('../../spec-helpers');
```

Comment:
@rafa-avila-bc if we are commenting out this whole file, is there a reason we still need it?

Reply:
I missed this. I already fixed it. Tests are now passing.